### PR TITLE
Minor changes to whats new

### PR DESCRIPTION
**Pull request by adegeo**

### CODE REVIEW

File: dotnet-desktop-guide/net/winforms/whats-new/index.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ There are a few breaking changes you should be aware of when migrating from .NET
 ## Enhanced features
 
 - Microsoft UI Automation patterns work better with accessibility tools like Narrator and Jaws.
-- Improved performance and reliability.
```
Comment (RussKie) [inline dotnet-desktop-guide/net/winforms/whats-new/index.md:17]: We did improve performance, so I think it is worth keeping. Just not "reliability" bit :)

**Review by RussKie — COMMENTED** [review]
👍

**Review by RussKie — APPROVED** [review]

File: dotnet-desktop-guide/net/winforms/whats-new/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ ms.date: 11/05/2020
 ms.topic: conceptual
 ---
 
-# What's new (Windows Forms.NET)
+# What's new (Windows Forms .NET)
```
Comment (tdykstra) [inline dotnet-desktop-guide/net/winforms/whats-new/index.md:8]: Could this just be "What's new for Windows Forms" given that you identify .NET 5 in the text?

**Review by tdykstra — APPROVED** [review]
LGTM

**Pull request merged**